### ISSUE TITLE
0.6.0 causes some unresolved dependencies

### ISSUE THREAD

**Issue by sosukesuzuki** [snowpackjs/rollup-plugin-polyfill-node#7]
Once update to `0.6.0` some unresolved dependencies are happened.

```
(!) Unresolved dependencies
https://rollupjs.org/guide/en/#warning-treating-module-as-external-dependency
polyfill-node:__zlib-lib/binding (imported by polyfill-node:zlib.js)
polyfill-node:_stream_transform (imported by polyfill-node:stream.js)
polyfill-node:_stream_duplex (imported by polyfill-node:stream.js)
polyfill-node:_stream_writable (imported by polyfill-node:stream.js)
polyfill-node:_stream_readable (imported by polyfill-node:stream.js)
polyfill-node:_stream_passthrough (imported by polyfill-node:stream.js)
```

Probably it caused by https://github.com/snowpackjs/rollup-plugin-polyfill-node/commit/866cba3c13c949fde4993f495a3f32c0c6e6adfa.

We are still using the form of `import { foo } from "polyfill-node:bar";` (e.g. https://github.com/snowpackjs/rollup-plugin-polyfill-node/blob/main/polyfills/zlib.js#L23) I think it should be updated to `import { foo } from "\0polyfill-node:bar";`.

**Comment by FredKSchott** [comment]
Ah you're right, thanks for catching!

**Comment by FredKSchott** [comment]
Fixed in v0.6.1

**Comment by sosukesuzuki** [comment]
Thank you!
